Formula/fontforge.rb

```diff
@@ -101,6 +101,18 @@ def install
     end
   end
 
+  def caveats
+    <<-EOS.undent
```

Comment:
@vitorgalvao Don't worry about changing but FYI @DomT4 I preferred the way before.